src/Illuminate/View/Compilers/BladeCompiler.php

```diff
@@ -277,7 +277,7 @@ protected function compileRawEchos($value)
 		{
 			$whitespace = empty($matches[3]) ? '' : $matches[3].$matches[3];
 
-			return $matches[1] ? substr($matches[0], 1) : '<?php echo '.$this->compileEchoDefaults($matches[2]).'; ?>'.$whitespace;
+			return $matches[1] ? substr($matches[0], 1) : '<?= '.$this->compileEchoDefaults($matches[2]).'; ?>'.$whitespace;
```

Comment:
> I still don't see it really add any benefit, unless this changes gain some huge performance.

Well, just as `[]` is not any more of benefits over `array()`. But Laravel has always been a modern framework, it should adapt the latest techniques whenever it can. I see quite a portion of L5 switched to `[]` instead of `array()`, `?:` instead of `? $var : $val`, so again, why not?